package.json

```diff
@@ -3,12 +3,19 @@
   "description": "Realtime client library for Ably.io, the realtime messaging service",
   "version": "0.9.0-beta.0",
   "main": "./nodejs/index.js",
+  "react-native": {
+    "./nodejs/index.js": "browser/static/ably-reactnative.js"
+  },
+  "browser": {
+    "./nodejs/index.js": "browser/static/ably-commonjs.js"
+  },
   "dependencies": {
     "async": "git://github.com/ably-forks/async#requirejs",
     "hexy": "~0.2",
     "msgpack-js": "git://github.com/paddybyers/msgpack-js.git",
     "request": "^2.74.0",
-    "ws": "^1.1.1"
+    "ws": "^1.1.1",
+    "react-native-randombytes": "git+https://github.com/ably-forks/react-native-randombytes.git"
```

Comment:
ah, that's unfortunate. We may end up going the dedicated-separate-library route eventually, but for now, s'probably fine -- at least it's only 188K, could be worse